packages/extension-chakra-storefront/src/components/list-menu/list-menu-content.jsx

```diff
@@ -76,7 +71,6 @@ const ListMenuContent = ({maxColumns, item, itemsKey, onClose, initialFocusRef})
                             links={links}
                             color={'gray.900'}
                             onLinkClick={onClose}
-                            {...(index === 0 ? {headingLinkRef: initialFocusRef} : {})}
```

Comment:
It seems that the first link is automatically focused, it works without this line...